src/pages/Home.js

```diff
@@ -69,6 +78,10 @@ const pointColourExpression = [
   '#27847A',
 ]
 
+String.prototype.format = function () {
+  return this.toLowerCase().replace(/\s/g, '_')
+}
```

Comment:
why are we changing string prototype?

Reply:
adding a format() functions, so not really changing, upgrading. To keep code cleaner. so I do not need to repeat this multiple times. Also creating a function i think it's an overkill, so added a new method to type String in this file. format()

Reply:
changed to a helper method.